src/macaron/repo_finder/repo_finder_enums.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2024 - 2025, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+"""This module contains Enums used to represent the outcome of Repo Finder or Commit Finder executions."""
+from enum import Enum
+
+
+class RepoFinderOutcome(Enum):
```

Comment:
I think this `enum` is not really the repo finder outcome, as in it doesn't contain the repo URL. 

```suggestion
class RepoFinderMetadata(Enum):
```